package.json

```diff
@@ -4,156 +4,64 @@
   "author": "Baruch Rothkoff <baruchiro@gmail.com>",
   "description": "An Electron-vue project for israeli-bank-scrapers",
   "license": "MIT",
-  "main": "./dist/electron/main.js",
+  "private": true,
   "scripts": {
-    "build": "node .electron-vue/build.js",
-    "build:release": "node .electron-vue/build.js && electron-builder",
-    "build:dir": "node .electron-vue/build.js && electron-builder --dir",
-    "build:clean": "cross-env BUILD_TARGET=clean node .electron-vue/build.js",
-    "build:web": "cross-env BUILD_TARGET=web node .electron-vue/build.js",
-    "dev": "node .electron-vue/dev-runner.js",
-    "e2e": "npm run pack && mocha test/e2e",
-    "lint": "eslint --ext .js,.vue -f ./node_modules/eslint-friendly-formatter src test",
-    "lint:fix": "eslint --ext .js,.vue -f ./node_modules/eslint-friendly-formatter --fix src test",
-    "pack": "npm run pack:main && npm run pack:renderer",
-    "pack:main": "cross-env NODE_ENV=production webpack --progress --colors --config .electron-vue/webpack.main.config.js",
-    "pack:renderer": "cross-env NODE_ENV=production webpack --progress --colors --config .electron-vue/webpack.renderer.config.js",
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "electron:build": "vue-cli-service electron:build",
```

Comment:
Where is the `vue-cli-service` configurations?

Reply:
[https://github.com/baruchiro/israeli-bank-scrapers-desktop/blob/a771ab23df06361d42b9752ae7e5c081d746fa31/package.json#L40-L44](https://github.com/baruchiro/israeli-bank-scrapers-desktop/blob/a771ab23df06361d42b9752ae7e5c081d746fa31/package.json#L40-L44)